mon-pix/tests/integration/components/signin-form-test.js

```diff
@@ -68,6 +68,29 @@ describe('Integration | Component | signin form', function() {
       expect(document.querySelector('a.sign-form-body__forgotten-password-link')).to.exist;
     });
 
+    it('should not display a Pole emploi button when Pole emploi connection is disabled', async function() {
```

Comment:
Je crois qu'il y a une duplication des tests Pôle Emploi dans ce fichier. J'ai mergé ce matin un ticket ou j'ai touché aussi au bouton Pôle Emploi et j'ai placé les deux tests dans un context 'when domain is pix.fr' ligne 156